### PR TITLE
Chritisbranch2.0

### PR DESCRIPTION
**Pull request by siemprechristie**

### CODE REVIEW

File: .github/workflows/test_runner.yml
Comment (brodyreiss1) [inline .github/workflows/test_runner.yml:1]: Was there a reason this had to change? It all looks good to me, just asking!

File: backend/app/routers/reviews.py
Comment (brodyreiss1) [inline backend/app/routers/reviews.py:1]: Looks perfect! Thanks for adding the documentation.

**Review by brodyreiss1 — APPROVED** [review]
Lgtm

**Review by haalkhaz — COMMENTED** [review]
These look good to me.

**Review by haalkhaz — APPROVED** [review]

**Pull request merged**